misc/userscripts/open_download

```diff
@@ -94,7 +95,7 @@ fi
 line=$(printf "%s\n" "${entries[@]}"  \
        | crop-first-column 55 \
        | column -s $'\t' -t \
-       | $ROFI_CMD "${rofi_default_args[@]}" $ROFI_ARGS) || true
+       | $ROFI_CMD "${rofi_default_args[@]}" "$ROFI_ARGS") || true
```

Comment:
one alternative might be to put the whole thing in quotes - `"$ROFI_CMD ${rofi_default_args[@]} $ROFI_ARGS"`. thoughts? else I'll just silence the warning.